src/quo/components/wallet/required_tokens/view.cljs

```diff
@@ -0,0 +1,66 @@
+(ns quo.components.wallet.required-tokens.view
+  (:require [quo.components.markdown.text :as text]
+            [quo.components.utilities.token.view :as token]
+            [quo.components.wallet.required-tokens.style :as style]
+            quo.theme
+            [react-native.core :as rn]
+            [schema.core :as schema]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:type {:optional true} [:enum :token :collectible]]
```

Comment:
To give more clarity on it, if we say `:type` is optional, we could also say what the default value is. 👍